server/webserver.go

```diff
@@ -703,7 +703,17 @@ func (s *LivepeerServer) StartWebserver() {
 		if s.LivepeerNode.Eth != nil && s.LivepeerNode.EthEventMonitor != nil {
 			activeEventSubMap := s.LivepeerNode.EthEventMonitor.EventSubscriptions()
```

Comment:
Should we check if the JobService is working in the same way that we are checking that the RewardService and RoundService are working?